components/tabBarComponents/EventTabBar.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import { Ionicons } from "@expo/vector-icons"
+import { BlurView } from "expo-blur"
+
+export const EventTabBar = () => {
+  const darkIconColor = "#1c1c1c"
+  const iconSize = 24
+
+  return (
+    <BlurView intensity={100} tint="light" style={styles.blur}>
+      <View style={styles.container}>
+        <TouchableOpacity style={styles.icon}>
+          <Ionicons name="location" size={iconSize} color={darkIconColor} />
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.icon}>
+          <Ionicons name="chatbox" size={iconSize} color={"grey"} />
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.icon}>
+          <Ionicons name="add-circle" size={60} color={darkIconColor} />
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.icon}>
+          <Ionicons name="notifications" size={iconSize} color={"grey"} />
```

Comment:
Use black with an opacity of say 0.35 instead of a gray color. It's easier to work with, and makes supporting things like dark mode easier in the future.